extensions/amp-consent/0.1/amp-consent.css

```diff
@@ -85,6 +85,13 @@ amp-consent.amp-hidden {
   animation: 1000ms amp-consent-ui-placeholder-spin linear infinite;
 }
 
+.i-amphtml-consent-alertdialog {
+  overflow: hidden;
```

Comment:
I ran a quick test and am still able to see the 1 by 1 dot?

Reply:
I had concerns about the SR not being able to pick up the pixel if it was offset, but I tested now and it seems fine.